code/__HELPERS/names.dm

```diff
@@ -241,3 +241,9 @@ GLOBAL_VAR(syndicate_code_response) //Code response for traitors.
 				. += "."
 			else
 				. += ", "
+
+/proc/moth_name(gender)

```

Comment:
I can't tell the difference between the names in each list so you should probably reconsider this proc.